### PR TITLE
GCW-2155 Fetch designations without associations

### PR DESCRIPTION
**Pull request by patrixr**
Make results on the listing smaller as suggested by Matt. We're hiding the body and keeping the header. 
We're also fetching the designations on that page without any associations to improve performance

### CODE REVIEW

File: app/templates/orders/order_block.hbs [hunk 1]
```diff
@@ -19,64 +19,66 @@
 
     </div>
 
-    <div class="row order_body">
+    {{#unless smallOrderBlocks}}
```
Comment (patrixr) [inline app/templates/orders/order_block.hbs]: The github diff looks a bit messy, but {{#unless smallOrderBlocks}} is the only thing I've added

File: app/templates/orders/order_block.hbs [hunk 2]
```diff
@@ -19,64 +19,66 @@
...
-              <span>{{order.dispatchedOrdersPackages.length}} <i class="fa fa-ship"></i></span>
+              {{order.stockitContact.fullName}}
             </div>
           </div>
-      {{/if}}
+        {{/if}}
 
-    </div>
+        {{#if (is-or order.designatedOrdersPackages.length order.dispatchedOrdersPackages.length)}}
+            <div class="row order_process_summary">
```
Comment (Darshan-Chauhan) [inline app/templates/orders/order_block.hbs]: Indentation

**Comment by Darshan-Chauhan** [comment]
@patrixr , we will need `excludeAssociations` for following lines also
- https://github.com/crossroads/stock.goodcity/blob/df98289615f126c2d4d51e91c31eba1151b238b6/app/routes/index.js#L5
- https://github.com/crossroads/stock.goodcity/blob/df98289615f126c2d4d51e91c31eba1151b238b6/app/components/focus-textfield.js#L43
- https://github.com/crossroads/stock.goodcity/blob/df98289615f126c2d4d51e91c31eba1151b238b6/app/routes/items/partial_designate.js#L14
- https://github.com/crossroads/stock.goodcity/blob/df98289615f126c2d4d51e91c31eba1151b238b6/app/routes/items/search_order.js#L59


Everything else looks fine

**Comment by patrixr** [comment]
@Darshan-Chauhan Updated according to comments above !

**Pull request merged**